diff -urN blivet-0.11.orig/blivet/__init__.py blivet-0.11/blivet/__init__.py
--- blivet-0.11.orig/blivet/__init__.py	2013-04-16 17:42:38.231934548 +0400
+++ blivet-0.11/blivet/__init__.py	2013-04-16 17:45:51.094489736 +0400
@@ -2732,7 +2732,11 @@
     except:
         relArch = None
 
-    filename = "%s/etc/redhat-release" % ROOT_PATH
+    filename = "%s/etc/rfremix-release" % ROOT_PATH
+
+    if not os.access(filename, os.R_OK):
+        filename = "%s/etc/redhat-release" % ROOT_PATH
+
     if os.access(filename, os.R_OK):
         with open(filename) as f:
             try:
