diff -urN blivet-0.10.orig/blivet/__init__.py blivet-0.10/blivet/__init__.py
--- blivet-0.10.orig/blivet/__init__.py	2013-04-12 19:55:47.628997839 +0400
+++ blivet-0.10/blivet/__init__.py	2013-04-12 19:58:51.986821270 +0400
@@ -2732,13 +2732,21 @@
     except:
         relArch = None
 
-    filename = "%s/etc/redhat-release" % ROOT_PATH
+    filename = "%s/etc/rfremix-release" % ROOT_PATH
     if os.access(filename, os.R_OK):
         with open(filename) as f:
             try:
                 relstr = f.readline().strip()
             except (IOError, AttributeError):
                 relstr = ""
+    else:
+        filename = "%s/etc/redhat-release" % ROOT_PATH
+        if os.access(filename, os.R_OK):
+            with open(filename) as f:
+                try:
+                    relstr = f.readline().strip()
+                except (IOError, AttributeError):
+                    relstr = ""
 
         # get the release name and version
         # assumes that form is something
