diff -urN blivet-0.58.orig/blivet/__init__.py blivet-0.58/blivet/__init__.py
--- blivet-0.58.orig/blivet/__init__.py	2014-06-26 18:46:13.598502594 +0400
+++ blivet-0.58/blivet/__init__.py	2014-06-26 18:47:44.047692860 +0400
@@ -3084,7 +3084,11 @@
     except OSError:
         relArch = None
 
-    filename = "%s/etc/redhat-release" % getSysroot()
+    filename = "%s/etc/rfremix-release" % getSysroot()
+
+    if not os.access(filename, os.R_OK):
+        filename = "%s/etc/redhat-release" % ROOT_PATH
+
     if os.access(filename, os.R_OK):
         (relName, relVer) = releaseFromRedhatRelease(filename)
     else:
