diff -urN blivet-1.0.2.orig/blivet/osinstall.py blivet-1.0.2/blivet/osinstall.py
--- blivet-1.0.2.orig/blivet/osinstall.py	2015-03-05 23:11:49.910300195 +0300
+++ blivet-1.0.2/blivet/osinstall.py	2015-03-05 23:13:54.357143574 +0300
@@ -124,7 +124,11 @@
     except OSError:
         relArch = None
 
-    filename = "%s/etc/redhat-release" % getSysroot()
+    filename = "%s/etc/rfremix-release" % getSysroot()
+
+    if not os.access(filename, os.R_OK):
+        filename = "%s/etc/redhat-release" % ROOT_PATH
+
     if os.access(filename, os.R_OK):
         (relName, relVer) = releaseFromRedhatRelease(filename)
     else:
