diff -urN blivet-0.43.orig/blivet/__init__.py blivet-0.43/blivet/__init__.py
--- blivet-0.43.orig/blivet/__init__.py	2014-03-05 14:34:22.614876902 +0400
+++ blivet-0.43/blivet/__init__.py	2014-03-05 14:35:28.007184362 +0400
@@ -3048,7 +3048,11 @@
     except:
         relArch = None
 
-    filename = "%s/etc/redhat-release" % ROOT_PATH
+    filename = "%s/etc/rfremix-release" % ROOT_PATH
+
+    if not os.access(filename, os.R_OK):
+        filename = "%s/etc/redhat-release" % ROOT_PATH
+
     if os.access(filename, os.R_OK):
         (relName, relVer) = releaseFromRedhatRelease(filename)
     else:
